docs/persona_paths/shop_manager.md

```diff
@@ -1,3 +1,7 @@
+---
+description: Perform tasks related to running an electronic presence of your store.
+---
+
 # Shop manager role
```

Comment:
It shouldn't be Store manager according to the discussion about the landing page tiles in the user-doc?